webapp/client/src/components/AnchorButton.test.js

```diff
@@ -35,4 +45,35 @@ describe("AnchorButton", () => {
       "download"
     );
   });
+
+  it("should show the AnchorSecondary when isSecondaryButton is true", () => {
+    const PROPS_DATA = { ...MOCK_PROPS, isSecondaryButton: true };
+    render(<AnchorButton {...PROPS_DATA} />);
+  });
+
+  it("should NOT show the AnchorSecondary when isSecondaryButton is true", () => {
+    const PROPS_DATA = { ...MOCK_PROPS, isSecondaryButton: false };
+    render(<AnchorButton {...PROPS_DATA} />);
+  });
+});
+
+describe("Anchor add plausible", () => {
+  const MOCK_PROPS = {
+    text: "anchor text",
+    url: "url/some/link/path",
+    name: "name/some/link/path",
+    plausibleDomain: "domain/some/link/path",
+    plausibleName: "plausible_name",
+  };
+
+  beforeEach(() => {
+    window.plausible = jest.fn().mockReturnValue({ plausible: jest.fn() });
+    render(<AnchorButton {...MOCK_PROPS} />);
+  });
+
+  it("should run the plausible function on click button", () => {
+    fireEvent.click(screen.getByText(MOCK_PROPS.text));
+
+    expect(window.plausible).toHaveBeenCalledWith("plausible_name", undefined);
+  });
```

Comment:
👍

Reply:
maybe in the future we can use the more common [user event testing library extension](https://testing-library.com/docs/ecosystem-user-event#api)

Reply:
Good remark. We use it for the other tests as well.